src/sentry/static/sentry/app/views/eventsV2/eventView.tsx

```diff
@@ -759,6 +760,10 @@ class EventView {
       delete payload.cursor;
     }
 
+    if (!payload.query) {
+      delete payload.query;
```

Comment:
Why? The tag facets should take the current conditions into account or you lose the ability to drill down into results.

Reply:
I'm reverting this. I'm going to put a revised payload shape for tags in a separate PR.